packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -102,7 +109,16 @@ export async function handleGetAttestationRequest(
   }
 
   const typedData = buildSecurityCodeTypedData(getRequest.account)
-  if (!verifyEIP712TypedDataSigner(typedData, authentication as string, getRequest.account)) {
+  const accounts = await useKit((kit) => kit.contracts.getAccounts())
+  const [walletAddress, dek] = await Promise.all([
+    accounts.getWalletAddress(getRequest.account),
+    accounts.getDataEncryptionKey(getRequest.account),
+  ])
+  if (
```

Comment:
This is a nit, but still will always do all signature verification against all addresses right? IMO we should short-circuit the authentication if one of them pass right?

Reply:
I think you've commented on an outdated commit but I see a few ways of doing it:

1. Preload all possible signers and verify signatures
2. Preload all possible signers and verify signatures (short circuiting on valid)
2. Lazily load accounts and short circuit on valid

I'm doing 2 right now, but 3 is also an option.

Reply:
Yeah agree on 2, I think I was commenting on an outdated commit